components/brave_wallet_ui/common/slices/endpoints/wallet.endpoints.ts

```diff
@@ -674,6 +674,15 @@ async function createDefaultAccounts({
             keyringIdForNewAccount(net.coin, net.chainId),
             suggestNewAccountName(accounts, net)
           )
+          return
+        }
+        case BraveWallet.CoinType.ZEC: {
+          await keyringService.addAccount(
+            net.coin,
+            keyringIdForNewAccount(net.coin, net.chainId),
+            suggestNewAccountName(accounts, net)
+          )
+          return
```

Comment:
Good find, I think we could actually do away with this `switch` here and then just modify the `if` statement above to exclude creating accounts for `ETH` and `SOL` since those are automatically being created in core. 

Maybe something like
```
      if (
        !allowedNewWalletAccountTypeNetworkIds.includes(
          networkEntityAdapter.selectId(net)
        ) ||
        net.coin === BraveWallet.CoinType.ETH ||
        net.coin === BraveWallet.CoinType.SOL
      ) {
        return
      }
      await keyringService.addAccount(
        net.coin,
        keyringIdForNewAccount(net.coin, net.chainId),
        suggestNewAccountName(accounts, net)
      )
```